azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/PackageMojo.java

```diff
@@ -250,7 +252,10 @@ protected void writeObjectToFile(final ObjectWriter objectWriter, final Object o
             throws IOException {
         targetFile.getParentFile().mkdirs();
         targetFile.createNewFile();
-        objectWriter.writeValue(targetFile, object);
+        final ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+        objectWriter.writeValue(byteArrayOutputStream, object);
+        // Change line of end to LF, refers https://stackoverflow.com/questions/3776923/how-can-i-normalize-the-eol-character-in-java
+        FileUtils.write(targetFile, byteArrayOutputStream.toString().replaceAll("\\r\\n?", "\n"));
```

Comment:
use a more elegant way of generating json file with correct line feeds.

Reply:
Change to use `DefaultIndenter` to do this job